pronto_mia/lib/core/queries/user_queries.dart

```diff
@@ -0,0 +1,59 @@
+class UserQueries {
+  static const users = """
+    query users() {
+      users {
+        id
+        userName
+        department {
+          id
+          name
+        }
+        accessControlList {
+          id
+          canEditDeploymentPlans
+          canEditUsers
+          canViewDeploymentPlans
+          canViewUsers
+        }
+      }
+    }
+  """;
+
+  static const createUser = """
+    mutation createUser(\$userName: String!, \$password: String!) {
+      createUser(
+        userName: \$userName,
+        password: \$password,
+        departmentId: 1,
+        accessControlList: {
+          canViewDeploymentPlans: true,
+		      canEditDeploymentPlans: true,
+		      canViewUsers: true,
+		      canEditUsers: true,
+		      canViewDepartments: true,
+		      canEditDepartments: true
+        }
+      ) {
+        id
+      }
+    }
+  """;
+
+  static const updateUser = """
+     mutation updateUser(\$id: Int!, \$userName: String!, \$password: String!) {
```

Comment:
Here you could also update the department as well as the acl.

Reply:
Does that mean when I update the User with a new department it is not added to the users list of the department?

Reply:
It is added. But in your operation there is no department neither a acl present which could be set.